README.md

```diff
@@ -50,7 +51,7 @@ For a more seemless local experience after the coredns is running, you can tempo
 
 ## Kubernetes
 
-To setup a kubernetes cluster, perform the following step: Go to the **kubernetes** directory and run `terraform init && terraform apply`
+To setup a kubernetes cluster, perform the following step: Go to the **kubernetes** directory and run `terraform init && terraform apply` (note that this operation is heavy on resources and will take several minutes)
```

Comment:
It will take several minutes. I think "heavy" on resources is a relative term. I didn't notice it on my machine.

Reply:
Ok sure I will remove the "heavy" part.

Reply:
Done.